ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/searchresults/resultitem-template.html

```diff
@@ -0,0 +1,11 @@
+<sly data-sly-template.resultItem="${@ model, count}">
+  <div class="result-item" result-item>
+    <a href="${model.url}">
+      <div class="result-meta">
+        <h3>${'{0}. {1}' @ format=[count,model.title], context='unsafe'}</h3>
```

Comment:
context unsafe? please change to at least `html`

Reply:
same as https://github.com/valtech-ch/saas-aem-module/pull/14#discussion_r705998039